pom.xml

```diff
@@ -328,7 +328,7 @@
             <dependency>
               <groupId>com.puppycrawl.tools</groupId>
               <artifactId>checkstyle</artifactId>
-              <version>6.14</version>
+              <version>6.15</version>
```

Comment:
http://checkstyle.sourceforge.net/releasenotes.html

Upgrade is optional, there was no change in the number of violations detected by the old and upgraded version.
I use IntelliJ checkstyle plugin and this plugin was upgraded to 6.15 checkstyle version. It is better to keep them in sync.